### PR TITLE
task1

### PR DESCRIPTION
**Pull request by M-Suzy**

### CODE REVIEW

File: src/com/egs/suzy/task1/FileHandler.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import java.io.*;
+
+public class FileHandler {
+    public static void CountHellos(File in, File out){
```
Comment (serghovhannisyan) [inline src/com/egs/suzy/task1/FileHandler.java]: please change the name of method to lowercase

File: src/com/egs/suzy/task1/TestCountHellos.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import java.io.File;
+import java.io.IOException;
+
+public class TestCountHellos {
+    public static void main(String[] args) throws IOException {
+        File hello = new File("hello.txt");
+        File result = new File("result.txt");
+        if(result.exists()){
+            result.delete();
+            result.createNewFile();
```
Comment (serghovhannisyan) [inline src/com/egs/suzy/task1/TestCountHellos.java]: I think creating new file is not necessary

File: src/com/egs/suzy/task1/FileHandler.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        try(BufferedReader reader = new BufferedReader(new FileReader(in));
+        BufferedWriter writer = new BufferedWriter(new FileWriter(out, true))){
+            String[] arr;
+            String line = reader.readLine();;
+            int count = 0, lineNum = 0;
+            while(line!=null){
```
Comment (serghovhannisyan) [inline src/com/egs/suzy/task1/FileHandler.java]: please use auto correction

File: src/com/egs/suzy/task1/FileHandler.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+              arr = line.split("\\s");
+              for(String s : arr){
+                  if(s.equalsIgnoreCase("hello"))  count++;
```
Comment (serghovhannisyan) [inline src/com/egs/suzy/task1/FileHandler.java]: even if there is a line of code after if, while, for etc, it still good to have braces { }

File: src/com/egs/suzy/task1/TestCountHellos.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import java.io.File;
+import java.io.IOException;
+
+public class TestCountHellos {
+    public static void main(String[] args) throws IOException {
+        File hello = new File("hello.txt");
```
Comment (serghovhannisyan) [inline src/com/egs/suzy/task1/TestCountHellos.java]: could we move .txt strings as a constant?

File: src/com/egs/suzy/task1/TestCountHellos.java (hunk 2 above)
Comment (M-Suzy) [reply src/com/egs/suzy/task1/TestCountHellos.java]: I wanted to have clear result file for each check and thought recreation of the new file is faster than clearing the content

**Review by serghovhannisyan — APPROVED** [review]

**Pull request merged**